SketchUp/length.rb

```diff
@@ -151,15 +151,15 @@ def >(length2)
   #   length1 = 11.to_l
   #   length2 = 12.to_l
   #   if length1 >= length2
-  #     puts "length1 is greater than or equal length2"
+  #     puts("length1 is greater than or equal length2")
```

Comment:
Same as above, and elsewhere, `puts` and `p` doesn't use parentheses by common convention.